commit a5857e8801806e58922bcce9dc3a4319cbad394a (refs/changes/48/2748/2)
Author: Bryan Cantrill <bryan@joyent.com>
Date:   2017-10-08T11:28:56+00:00 (2 years ago)
    
    OS-6385 zfs_arc_overflow_shift needs sufficient margin for ARC reclaim
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Approved by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/usr/src/uts/common/fs/zfs/arc.c b/usr/src/uts/common/fs/zfs/arc.c
index 5b58379915..fe60c09daa 100644
--- a/usr/src/uts/common/fs/zfs/arc.c
+++ b/usr/src/uts/common/fs/zfs/arc.c
@@ -303,7 +303,7 @@ int zfs_arc_evict_batch_limit = 10;
 static int		arc_grow_retry = 60;
 
 /* shift of arc_c for calculating overflow limit in arc_get_data_impl */
-int		zfs_arc_overflow_shift = 8;
+int		zfs_arc_overflow_shift = 3;
 
 /* shift of arc_c for calculating both min and max arc_p */
 static int		arc_p_min_shift = 4;
